auto-generated/cluster/stone-prd-rh01/tenants/crt-redhat-acm-tenant/appstudio.redhat.com_v1alpha1_component_maestro-main.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: appstudio.redhat.com/v1alpha1
+kind: Component
+metadata:
+  annotations:
+    appstudio.openshift.io/pac-provision: request
+    build.appstudio.openshift.io/request: configure-pac
+    image.redhat.com/generate: "true"
+  name: maestro-main
```

Comment:
Do we want the metaname of the component to be identical to the application?

Reply:
I debated that but ultimately - yeah in this case since it's standalone